cpu/native/Makefile.dep

```diff
@@ -1,6 +1,12 @@
-ifneq (,$(filter periph_spi,$(USEMODULE)))
-  USEMODULE += periph_spidev_linux
+ifeq ($(OS),Linux)
+  ifneq (,$(filter periph_gpio,$(USEMODULE)))
+    DEFAULT_MODULE += periph_gpio_linux
```

Comment:
```suggestion
    USEMODULE += periph_gpio_linux
```
I would not use DEFAULT_MODULE for this.

If it's because of the failing tests, then in the tests, instead of using DISABLE_MODULE, add native to CI_BOARD_BLACKLIST.

Reply:
I guess it could still be useful if someone wants to use the mock GPIO implementation instead.

Reply:
How is the mock named?, maybe something like this can be used?

```suggestion
    ifneq (,$(filter periph_gpio_mock,$(USEMODULE)))
     USEMODULE += periph_gpio_linux
    endif
```

Reply:
The mock is just `periph_gpio`.
I can make that a pseudomodule and move the implementation to `gpio_mock.c`.

Then in the tests we can do

```Makefile
ifeq (native,$(BOARD))
  USEMODULE += periph_gpio_mock
endif
```